src/cheri/zylevels1.adoc

```diff
@@ -0,0 +1,171 @@
+[#section_zylevels1,reftext="Zylevels1"]
+== "Zylevels1" Extension for CHERI 2-Level Information Flow Control
+
+Zylevels1 introduces a simple _Information Flow Control_ (IFC) mechanism to CHERI.
+It is mutually exclusive with <<section_zylevelsN>>, which introduces more elaborate IFC.
+
+In this simple IFC system, capabilities are said to be either _global_ or _local_.
+While the distinction between _global_ and _local_ is not one of _authority_
+(that is, the distinction is not one of _permission_),
+_global_ capabilities may attenuate into _local_ capabilities.
+The distinction refines the behavior of capability store and load instructions:
+
+* Capability-write-permissive capabilities are refined to authorize stores of _any_ capability or _global_ capabilities only.
+  The former may attenuate into the latter.
+  Attempting to store a local capability through an insufficiently permissive authority clears the tag of the value written to memory, if any.
+
+* Capability-load-permissive capabilities are refined to authorize loads of _any_ capabilities or _local_ capabilities only.
+  Again, the former may attenuate to the latter.
+  Attempting to load a global capability through an insufficiently permissive authority instead _attenuates_ the load result as described below.
+
+=== Added Architectural Permissions (AP) Bits
+
+[#zylevels1_ap_field_summary,width="100%",options=header,halign=center,cols="2,2,5"]
+|==============================================================================
+| Permission   | Type | Comment
+| <<zylevels1_sl_perm>>  | Data memory permission         | Used to filter the permissions of loaded capabilities.
+| <<zylevels1_lg_perm>>  | Data memory permission         | Used to filter the permissions of loaded capabilities.
+|==============================================================================
+
+[#zylevels1_sl_perm,reftext="SL-permission"]
+Store Local Permission (SL):: This field allows limiting the propagation of _local_ capabilities.
++
+A capabilty without <<zylevels1_gl_perm>> set stored through an authorizing capability lacking <<zylevels1_sl_perm>> will be stored with a zero valid tag.
+
+<<zylevels1_sl_perm>> is a _refinement of_ <<c_perm>> and <<w_perm>>.
+That is, if either of the latter are clear, then <<zylevels1_sl_perm>> has no effect.
+The capability encoding might take advantage of this dependence for compression,
+and so clearing either <<c_perm>> or <<w_perm>> may result in
```

Comment:
I stumbled upon the "may result in". Would the spec need to be precise on whether SL will be cleared or not?